Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Display/XamlRenderer.cs

```diff
@@ -986,24 +987,33 @@ private void RenderItalicRun(InlineCollection inlineCollection, ItalicTextInline
         /// <param name="context"> Persistent state. </param>
         private void RenderStrikethroughRun(InlineCollection inlineCollection, StrikethroughTextInline element, RenderContext context)
         {
-            Span span = new Span
+            Span span = new Span();
+
+            if (ApiInformation.IsTypePresent("Windows.UI.Text.TextDecorations"))
             {
-                FontFamily = new FontFamily("Consolas")
-            };
+                span.TextDecorations = TextDecorations.Strikethrough;
+            }
+            else
+            {
+                span.FontFamily = new FontFamily("Consolas");
+            }
 
             // Render the children into the inline.
             RenderInlineChildren(span.Inlines, element.Inlines, span, context);
 
             AlterChildRuns(span, (parentSpan, run) =>
             {
-                var text = run.Text;
-                var builder = new StringBuilder(text.Length * 2);
-                foreach (var c in text)
+                if (!ApiInformation.IsTypePresent("Windows.UI.Text.TextDecorations"))
```

Comment:
What about putting this into a static private property to avoid checking numerous times.